src/interfaces/chain.cpp

```diff
@@ -347,6 +341,7 @@ class ChainImpl : public Chain
     CAmount maxTxFee() override { return ::maxTxFee; }
     bool getPruneMode() override { return ::fPruneMode; }
     bool p2pEnabled() override { return g_connman != nullptr; }
+    bool isReadyToBroadcast() override { return !::fImporting && !::fReindex && !IsInitialBlockDownload(); };
```

Comment:
Nit: Extra `;` at end of line :-)

Reply:
Thanks @practicalswift. That commit is from https://github.com/bitcoin/bitcoin/pull/15632, where I've now removed the extraneous `;`. Do you mind reviewing that PR?